import_package.md

```diff
@@ -0,0 +1,280 @@
+# Import package
+
+* An import package is a .zip file containing a **manifest.yaml** that describes additional 
+  contents and actions that should be carried out when processing the package.
+* A valid import package must contain all resource files that are referenced in the 
+  **manifest.yaml**.
+
+## Meta-data
+
+* `apiVersion` is set to `v1beta1` 
+
+## Tasks
+
+* `tasks`: is a list of api actions and resource tasks.
+    * `id`: Each task must contain a unique **id** property that only contains upper-case, 
+       lower-case letters, digits and an underscore (`[A-Za-z0-9_]+`)
```

Comment:
```suggestion
       lower-case letters, digits or underscores (`[A-Za-z0-9_]+`)
```
For a spec, "an underscore" implies that the name can only contain a single underscore.  Is that what you want?